ui/app/src/Main.elm

```diff
@@ -86,12 +87,25 @@ init flags url key =
 
             else
                 "/"
+
+        firstDayOfWeek =
+            flags
+                |> Json.decodeValue (Json.field "firstDayOfWeek" Json.string)
+                |> Result.withDefault "Monday"
```

Comment:
I'm sorry, I misunderstood the original issue, that `Sunday` indeed used to be the default.

Reply:
no problem, since changing the default is now a single line change.